executor/ddl_test.go

```diff
@@ -1131,65 +1131,6 @@ func TestAutoRandomClusteredPrimaryKey(t *testing.T) {
 	tk.MustQuery("select a from t;").Check(testkit.Rows("1"))
 }
 
-// Test filter different kind of allocators.
-// In special ddl type, for example:
-// 1: ActionRenameTable             : it will abandon all the old allocators.
-// 2: ActionRebaseAutoID            : it will drop row-id-type allocator.
-// 3: ActionModifyTableAutoIdCache  : it will drop row-id-type allocator.
-// 3: ActionRebaseAutoRandomBase    : it will drop auto-rand-type allocator.
-func TestFilterDifferentAllocators(t *testing.T) {
```

Comment:
Updated and moved to https://github.com/pingcap/tidb/pull/38808/files#diff-1c27a2956e8d617d4597580c00fb81d42bf81a4a7e726d911e11ac25ccb1e484R37